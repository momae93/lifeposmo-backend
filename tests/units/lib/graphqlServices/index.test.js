const graphQLLib = require('../../../../src/lib/graphqlServices');

describe('[LIB][GRAPHQL] : graphql lib', () => {
  it('should match snapshot from graphql lib', async () => {
    expect(graphQLLib).toMatchSnapshot();
  });
});
