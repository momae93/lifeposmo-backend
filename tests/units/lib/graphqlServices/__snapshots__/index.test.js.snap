// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[LIB][GRAPHQL] : graphql lib should match snapshot from graphql lib 1`] = `
Object {
  "combineResolvers": [Function],
  "combineSchemas": [Function],
  "constants": Object {
    "SCALAR_DATE": "Date",
  },
}
`;
